### PR TITLE
Keyfile support for ChangePassphrase

### PR DESCRIPTION
**Pull request by coldfix**
As discussed in #184.

BTW: Should we also allow adding/removing passphrases?

### CODE REVIEW

**Comment by vpodzime** [comment]
> BTW: Should we also allow adding/removing passphrases?

I'm not aware of any request for such functionality, but it would definitely be a natural one. :)

File: data/org.freedesktop.UDisks2.xml [hunk 1]
```diff
@@ -1800,10 +1800,14 @@
         ChangePassphrase:
         @passphrase: The existing passphrase.
         @new_passphrase: The new passphrase to use.
-        @options: Options (currently unused except for <link linkend="udisks-std-options">standard options</link>).
+        @options: Options - known options (in addition to <link linkend="udisks-std-options">standard options</link>) includes <parameter>old_keyfile_contents</parameter> and <parameter>old_keyfile_contents</parameter> (of type 's' or 'ay').
 
         Changes the passphrase to @new_passphrase. An existing passphrase is required.
 
+        If <parameter>old_keyfile_contents</parameter> or
+        <parameter>old_keyfile_contents</parameter> are given, they take
```
Comment (vpodzime) [inline data/org.freedesktop.UDisks2.xml]: ``old_keyfile_contents`` mention twice here. And the same applies to the ``@options...`` line above.

Comment (coldfix) [inline data/org.freedesktop.UDisks2.xml]: oops. corrected

**Comment by vpodzime** [comment]
@coldfix, could you please squash the last commit into the first one? Or maybe even all three into a single one adding the feature + docs + tests? Thanks!

**Comment by coldfix** [comment]
Done.

**Pull request merged**